src/google/protobuf/map_unittest.proto

```diff
@@ -61,6 +61,11 @@ message TestMap {
   map<int32   , ForeignMessage> map_int32_foreign_message = 17;
   map<string  , ForeignMessage> map_string_foreign_message = 18;
   map<int32   , TestAllTypes> map_int32_all_types = 19;
+
```

Comment:
Are you able to reproduce the error by current .proto files? I guess google.protobuf.unittest_pb2.TestAllTypes should be good enough.

Reply:
Oops, good catch. The old PR needed this, the new one creates the bug just fine without it. :)